### ISSUE TITLE
Move config/database.yml.j2 to the app

### ISSUE THREAD

**Issue by sauloperez** [openfoodfoundation/ofn-install#387]
## Description

Rails app's way to configure the DB connection details is through the `config/database.yml` file or `ENV['DATABASE_URL']`. See "3.3 Configuring a Database" in https://guides.rubyonrails.org/v3.2/getting_started.html for more details.

As detailed in https://github.com/openfoodfoundation/ofn-install/issues/97, as happens with initializer it makes no sense to have a separate `database.yml` in the app's repo and another one in `ofn-install`. This is, as we have experienced in other situations, unnecessarily complex and error-prone. While looking at the app's yml file you may think that is what gets used in production while it is not.

I strongly believe it was implemented this way because no one realized this what ENV vars are meant for.

## Proposed solution

* Extract the app's `database.yml` values as ENV vars defined in `application.yml`
* Populate `postgresql.yml.j2` ansible vars as ENV vars in `application.yml`
* Make the app's `database.yml` read from ENV, providing current values as defaults
* Remove `postgresql.yml.j2` and its ansible tasks.
* Celebrate the greatness of simplicity :tada:

**Comment by sigmundpetersen** [comment]
From #483
> This will address #387

Guess we can close @sauloperez ?

**Comment by sauloperez** [comment]
Not really. I missed a _help_ in that quote. We still need to work on the last two items of the proposed solution.

**Comment by mkllnk** [comment]
No problem to solve here. We can clean up if we work on this part of the code.
